pymodbus/client/mixin.py

```diff
@@ -59,11 +59,13 @@ def execute(
         .. tip::
             Response is not interpreted.
         """
-        return ModbusResponse()
+        raise NotImplementedError(
```

Comment:
Have you considered using Abstract Base Classes?  https://docs.python.org/3/library/abc.html

Reply:
Never mind, it annoys `pylint` and I don't feel like suppressing those warnings.